tests/Elastic.Ingest.Elasticsearch.Tests/ElasticsearchChannelTests.cs

```diff
@@ -61,19 +61,18 @@ public void BackoffRetries()
 		session.TotalBulkRequests.Should().Be(5);
 		session.TotalRetries.Should().Be(3);
 		session.Rejections.Should().Be(0);
-
 	}
 
 	[Fact]
 	public void BackoffTooMuchEndsUpOnDLQ()
 	{
 		var client = TestSetup.CreateClient(v => v
-				// first two events keep bouncing
+			// first two events keep bouncing
 			.ClientCalls(c => c.BulkResponse(429, 429))
 			.ClientCalls(c => c.BulkResponse(429, 429)) //retry 1
 			.ClientCalls(c => c.BulkResponse(429, 429)) //retry 2
 			.ClientCalls(c => c.BulkResponse(429, 429)) //retry 3
-				// next two succeed straight away
+														// next two succeed straight away
```

Comment:
nitpick but reformat will move this back atleast for me on Rider.

Reply:
VS format is the one sending those over to align with the others! 😠 

I'll fix them up manually!